src/js/components/landing/recentActivity/SubmissionLink.jsx

```diff
@@ -18,10 +19,25 @@ const defaultProps = {
 }
 
 export default class SubmissionLink extends React.Component {
+	constructor(props) {
+		super(props);
+		this.state = {
+			page: '#/'
```

Comment:
Good point, empty string works, actually i think it may be better to go to a 404 because the only time that will happen is when the api returns a '0' which means the submission files do not exist